### ISSUE TITLE
Whiteboard color picker is not accessible via keyboard

### ISSUE THREAD

**Issue by dariiing** [cryptpad/cryptpad#1823]
The color picker is not accessible for keyboard users:

![Image](https://github.com/user-attachments/assets/5884c83b-0ebc-498c-a00e-193395928bf4)
